config/pipelines/high_throughput_wgs.yml

```diff
@@ -19,5 +19,6 @@ WGS MX: # Bottom of the pipeline (Pooling and normalization)
     request_type_key:
       - limber_multiplexing
   relationships:
+    Library Stock Plate: LB Lib Pool #These are supplied as ready-made libraries and enter the pipeline at this point.
```

Comment:
If the MX stage follows the lib prep stage, how is this new plate be linked to the lib prep stage?